pkg/config/types/portrange.go

```diff
@@ -1,161 +0,0 @@
-package types
```

Comment:
what's the deal with this, was this just not used?

Reply:
Yep it's dead code that was left over